### PR TITLE
Fixes backlink to earlier chapter

### PR DESCRIPTION
**Pull request by hlapp**
I actually am not sure that this is indeed the correct fix (it seems this is what other chapters with back or forward references do). The original link, however, results in a 404.

### CODE REVIEW

File: 03-what-to-teach.Rmd [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Deciding what to teach
 
-As discussed in an [earlier chapter](https://carpentries.github.io/curriculum-development/conceptual-elements.html#backward-design), the first step in designing a curriculum according to backward design principles is to
+As discussed in an [earlier chapter][Backward design], the first step in designing a curriculum according to backward design principles is to
```
Comment (tobyhodges) [inline 03-what-to-teach.Rmd:3]:
```suggestion
As discussed in an [earlier chapter](https://cdh.carpentries.org/guiding-principles.html#backward-design), the first step in designing a curriculum according to backward design principles is to
```

The link points to an anchor in a specific page, this adds a direct link with the correct URL.

**Comment by tobyhodges** [comment]
Thank you for bring this up, @hlapp. I tried building the pages locally and it did not seem like your approach worked (perhaps because the _Backwards Design_ section is an anchor within a larger page?) but I have suggested the corrected URL. If you commit that suggestion, I will merge this and that should fix the link.

**Comment by hlapp** [comment]
@tobyhodges do you know where we could find the piece of template or code that renders the link? Perhaps there is actually a mechanism for linking to in-page anchors. Though I suspect you're right that there isn't.

**Comment by tobyhodges** [comment]
I did not previously know. But your question sent me looking and, indeed, the Bookdown documentation suggests that the kind of cross-referencing you were trying to use _should_ work: https://bookdown.org/yihui/bookdown/cross-references.html

However, the change you made definitely did not in my local build. I have just tried using the `[earlier chapter](#backward-design)` approach, and that _did_ work! I will update my suggested change again to reflect this.

**Comment by hlapp** [comment]
Awesome @tobyhodges, just committed it.

**Comment by tobyhodges** [comment]
Thank you @hlapp

**Pull request merged**